ux.symfony.com/LICENSE

```diff
@@ -0,0 +1,19 @@
+Copyright (c) 2022-present Fabien Potencier
```

Comment:
Note: adding this for clarity, but the entire repository has a root LICENSE file. And as ux.symfony.com is never a sub-tree split, this never lives on its own. But adding it just in case.